tesseract_task_composer/core/include/tesseract_task_composer/core/task_composer_pipeline.h

```diff
@@ -33,12 +33,12 @@ TESSERACT_COMMON_IGNORE_WARNINGS_PUSH
 TESSERACT_COMMON_IGNORE_WARNINGS_POP
 
 #include <tesseract_task_composer/core/task_composer_graph.h>
+#include <tesseract_task_composer/core/task_composer_node_info.h>
```

Comment:
I this necessary? Forward declaration should be good because it is only used in a return type.

Reply:
MSVC seems to have a slightly different method to handle std::unique_ptr that needs the full definition. Otherwise I get a "cannot delete incomplete type" build error.